applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -62,4 +67,32 @@
         </div>
         @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
     }
+
+    <script>
+        require(['Promise'], function (Promise) {
```

Comment:
Hmm I think that the Promise module is always bundled in PROD so it probably wouldn't even be available, @SiAdcock do you know if this is the case?